packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -0,0 +1,855 @@
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/devices/guo4a5u7JHHhjXrMK0g4"
+```
+
+##### Response
+
+```json
+{
+    "id": "ftrZooGoT8b41iWRiQs7",
+    "status": "CREATED",
+    "created": "2019-10-02T18:03:07.000Z",
+    "lastUpdated": "2019-10-02T18:03:07.000Z",
+    "profile": {
+        "displayName": "Example Device name",
+        "platform": "WINDOWS",
+        "manufacturer": null,
+        "model": null,
+        "osVersion": null,
+        "serialNumber": "Example",
+        "imei": null,
+        "meid": null,
+        "udid": null,
+        "sid": "S-1-11-111"
+    },
+    "_links": {
+        "activate": {
+            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7JHHhjXrMK0g4/lifecycle/activate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7JHHhjXrMK0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PATCH",
+                    "PUT"
+                ]
+            }
+        },
+        "devices": {
+            "href": "https://{yourOktaDomain}/api/v1/devices/guo4a5u7JHHhjXrMK0g4/devices",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+##### Error Response
+
+An invalid `id` returns a `404 Not Found` status code.
+
+```http
+HTTP/1.1 404 Not Found
+Content-Type: application/json
+
+{
+    "errorCode": "E0000007",
+    "errorSummary": "Not found: Resource not found: 123456 (GenericUDObject)",
+    "errorLink": "E0000007",
+    "errorId": "oaeksGoibBmS9OGYo4vXT7llA",
+    "errorCauses": []
+}
+```
+
+### List Devices
+
+<ApiOperation method="get" url="/api/v1/devices" />
+
+Fetches a list of all Devices that are not `DELETED` for your org. Responses will be paginated with maximum size of 200.
+
+A subset of Devices can be returned that match a supported search criteria using the `search` query parameter.
+
+Searches for devices based on the properties specified in the `search` parameter conforming SCIM filter specifications (case insensitive). This data will be eventually consistent. Different results are returned depending on specified queries in the request. Empty list will be returned if no objects match `search` request.
+
+> **Note:** Listing devices with `search` should not be used as a part of any critical flows, like authentication, updates etc. to prevent potential data loss. `search` results may not reflect latest information, as this endpoint uses a search index which may not be up to date with recent updates to the object. <br> Don't use search results directly for record updates, as the data might be stale and therefore overwrite newer data, resulting in data loss. <br> Use an `id` lookup for records that you update to ensure your results contain the latest data.
```

Comment:
```suggestion
> **Note:** Listing devices with `search` should not be used as a part of any critical flows—such as authentication or updates—to prevent potential data loss. `search` results may not reflect the latest information, as this endpoint uses a search index which may not be up-to-date with recent updates to the object. <br> Don't use search results directly for record updates, as the data might be stale and therefore overwrite newer data, resulting in data loss. <br> Use an `id` lookup for records that you update to ensure your results contain the latest data.
```